nnvm/python/nnvm/frontend/pytorch/aten.py

```diff
@@ -0,0 +1,603 @@
+r'''This file contains one class per PyTorch ATen operator. For the full
+list of operators, see
+https://github.com/zdevito/ATen/blob/master/aten/src/ATen/native/native_functions.yaml
+'''
+import numpy as np
+import tvm
+from nnvm.compiler.graph_util import infer_shape
+from nnvm.symbol import Variable, Symbol
+from nnvm.graph import create
+from nnvm.frontend.common import get_nnvm_op
+from .base import PyTorchOp
+
+
+class ATenOp(PyTorchOp): # pylint: disable=invalid-name
```

Comment:
Having to add `# pylint: disable=invalid-name` is not a good sign. It can not avoided by adding a dictionary in `aten.py` and `prim.py` that maps operator name to operator class.

Reply:
I chose this design because adding a dictionary would be somewhat redundant and because I don't want to have to think about what to name the classes here. With names matching ATen and Prim, it's clear what each operator maps to. Nonetheless, I'm ok with adding the dictionary.

Reply:
@KoinFlyp Current dynamic lookup method seems convenient, but it is actually a big dependency on PyTorch's naming convention of operators. For example right now their operator name is not following Python class naming convention. All TVM's other framework converters maintain a map, let's be consistent with other converters. Also it's clear to check what are all the supported operators from the dictionary.